File: Quiz_Dart/QuestionModel.dart
// Question Model
class Question{

  late String question ;
  late List<dynamic> option;
  late String answer;
  late List<dynamic> ans;

  Question(String question , List<dynamic> option , String answer){

    this.question =question;
    this.option=option;
    this.answer=answer;
  }
  Question.overloaded(String question ,List<dynamic> option , List<dynamic> ans){

    this.question =question;
    this.option=option;
    this.ans=ans;
  }
}

File: Quiz_java/QuestionModel.java
package Quiz;

public class QuestionModel {

   String question ;
   String[] option;
   String answer;
   String[] ans;

  QuestionModel(String question , String[] option , String answer){

    this.question =question;
    this.option=option;
    this.answer=answer;
  }
  
  QuestionModel(String question ,String[] option , String[] ans){

    this.question =question;
    this.option=option;
    this.ans=ans;
  }

}